pygsti/tools/matrixtools.py

```diff
@@ -925,12 +980,13 @@ def random_hermitian(dim):
         dim = int(dim)
         a = _np.random.random(size=[dim, dim])
         b = _np.random.random(size=[dim, dim])
-        c = a + 1.j * b + (a + 1.j * b).conj().T
+        c = a + 1.j * b
+        c += c.conj().T
         my_norm = norm1(c)
     return c / my_norm
 
 
-def norm1to1(operator, num_samples=10000, mx_basis="gm", return_list=False):
+def norm1to1(operator, num_samples=10000, mx_basis="gm"):
```

Comment:
This function was never called with ``return_list=True``, so I removed that option. The efficiency improvements are very minor.